crates/ra_ide/src/ssr.rs

```diff
@@ -72,6 +70,7 @@ type Binding = HashMap<Var, SyntaxNode>;
 struct Match {
     place: SyntaxNode,
     binding: Binding,
+    ignored_comments: Vec<SyntaxToken>,
```

Comment:
We have a strongly-typed Comment here, let's use this type?

https://github.com/rust-analyzer/rust-analyzer/blob/5e6f4ca6905e02a5fa34ef292fa2abefcd982222/crates/ra_syntax/src/ast/tokens.rs#L10

Reply:
Done!